examples/segment-anything-client/vite.config.js

```diff
@@ -1,6 +1,17 @@
 import { defineConfig } from 'vite';
-export default defineConfig({
-  build: {
-    target: 'esnext'
+export default defineConfig(env => {
+  const config = {
+    build: {
+      target: 'esnext'
+    }
+  };
+
+  if (env.mode === 'development') {
```

Comment:
Vite works perfect in production mode (`vite build`)

Reply:
thanks!